icons/wasm/wasm-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M.063 63.777V0h49.156l.004.91c.02 3.191 1.152 6.352 3.203 8.941.531.676 1.68 1.813 2.359 2.34a14.84 14.84 0 0 0 6.602 2.934c.727.129 1.023.148 2.453.145 1.734 0 2.242-.055 3.504-.379 2.742-.699 4.949-1.969 6.93-3.996 1.207-1.234 2.027-2.406 2.77-3.961.887-1.859 1.313-3.641 1.383-5.801L78.465 0h49.152v127.555H.063zm47.867 50.047a5795.98 5795.98 0 0 0 3.344-15.281l3.379-15.27c.043-.047.074-.039.098.023.02.051 1.418 6.949 3.113 15.332l3.078 15.246 4.184.016c3.316.012 4.191 0 4.219-.066.121-.32 10.703-44.902 10.672-44.977-.031-.086-.539-.098-4.082-.098-2.754 0-4.062.023-4.09.066s-1.516 7.016-3.312 15.504l-3.312 15.473c-.023.023-.066.016-.094-.02s-1.441-6.961-3.137-15.398l-3.133-15.48-.051-.145h-3.883c-3.031 0-3.898.02-3.945.082-.035.043-1.617 6.934-3.516 15.313l-3.484 15.27c-.02.02-.062.012-.094-.023s-1.336-6.875-2.902-15.207l-2.906-15.289-.059-.145h-4.109c-3.211 0-4.121.02-4.168.082-.043.055 1.496 7.297 4.766 22.477l4.867 22.5c.035.086.527.098 4.285.098 3.336 0 4.25-.02 4.273-.082zm39.313-.156c.035-.133.512-2.277 1.059-4.77l1.055-4.766.066-.242 7.602.016 7.602.02 1.402 4.879 1.449 4.988c.043.105.313.113 4.387.113 3.059 0 4.34-.02 4.34-.07 0-.172-13.227-44.996-13.289-45.035-.043-.027-3.02-.047-6.621-.047-5.371-.004-6.555.012-6.613.082-.07.082-10.914 44.762-10.914 44.965 0 .094.359.105 4.207.105h4.207zm3.855-17.266c-.027-.066 3.414-15.77 3.586-16.379.031-.102.172-.109 1.813-.109s1.777.008 1.809.109c.203.641 4.18 16.363 4.152 16.41-.023.039-2.164.063-5.68.063-4.84 0-5.648-.016-5.68-.094zm0 0"/></svg>
```

Comment:
As far as I can tell this is the same icon as `wasm-original`, but without the color. Which means we could remove this one, and put in an alias from `original` to  `plain` instead. Saving a tiny bit of space :)

I also made a suggestion in the `devicon-json` adding in the alias.

Reply:
plain file is deleted and alias used